contributor-guide/modules/ROOT/pages/docs/documentation-guidelines.adoc

```diff
@@ -1,73 +1,581 @@
-= Documentation Guidelines
-:navtitle: Documentation Guidelines
+= Guidelines
```

Comment:
In the rendered output, the code blocks say "bash" instead of "asciidoc"?

Reply:
good catch - changed to asciidoc